assets/js/atomic/components/product-list/summary/index.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+const ProductListSummary = ( { className, product } ) => {
+	if ( ! product.description ) {
+		return null;
+	}
```

Comment:
Since this is only using the product description, I would suggest just receiving the description as a prop.